### ISSUE TITLE
Fix conversion of glTF with external buffers

### ISSUE THREAD

**Issue by xeolabs** [xeokit/xeokit-convert#25]
````convert2xkt```` glTF->XKT conversion mode breaks when the glTF file provides its geometry in a separate binary file attachment.

This can be reproduced by running the open source converter tools like this:

````bash
IfcConvert-v0.6.0-517b819-linux64/IfcConvert ./myModel.ifc ./myModel.dae

COLLADA2GLTF/build/COLLADA2GLTF-bin -i ./myModel.dae -o ./myModel.gltf -s 

convert2xkt.js -s ./myModel.gltf -o ./myModel.xkt -l
````

````convert2xkt```` uses  ````fs.readFileSync```` to load the binary file, and expects the loaded data to be an ````ArrayBuffer````. However, that function returns a ````Buffer````. 

Solution is to simply convert the ````Buffer```` to an ````ArrayBuffer````.

**Comment by Amoki** [comment]
COLLADA2GLTF is a big library. The same can be done in 20 lines of python without dependencies. I could give you those lines :)

**Comment by jenbuzz** [comment]
"convert2xkt will automatically resolve that separate buffer file while it parses the glTF JSON file." - does this already happen or would that be an upcoming feature?

**Comment by xeolabs** [comment]
@jenbuzz already happens - give it a shot with your big files and see how it goes

**Comment by jenbuzz** [comment]
@xeolabs didn't go well. Tried multiple files but when running convert2xkt I now get an "JavaScript heap out of memory" error (on a machine with 65gb memory). Any ideas?

**Comment by Amoki** [comment]
Probably related to https://futurestud.io/tutorials/node-js-increase-the-memory-limit-for-your-process

**Comment by Amoki** [comment]
What command did you try?

**Comment by jenbuzz** [comment]
@Amoki "node --max-old-space-size=8192 convert2xkt.js ..." (with different size values)

**Comment by jenbuzz** [comment]
@Amoki well, it is killed when server runs out of free memory (and that's always beyond specified limit not matter what it is). Maybe my larger files are still too large for this to work unless an insane amount of memory is available. But with a gltf file around ~100mb and a bin file around 350mb I thought it might.

**Comment by tmarti** [comment]
Mitigated by https://github.com/xeokit/xeokit-convert/pull/27 😄

**Comment by xeolabs** [comment]
Fix published in @xeokit/xeokit-convert 1.0.8-beta-5
